katas/content/superdense_coding/index.md

```diff
@@ -0,0 +1,30 @@
+# Superdense Coding Kata
+
+@[section]({
+    "id": "./superdense_coding.ipynb__overview",
+    "title": "Overview"
+})
+
+**Superdense Coding** quantum kata is a series of exercises designed to get you familiar with programming in Q#.
+
+It covers the superdense coding protocol which allows us to transmit two bits of classical information by sending just one qubit using previously shared quantum entanglement.
+
+- A good description can be found in [the Wikipedia article](https://en.wikipedia.org/wikiSuperdense_coding).
+- A great interactive demonstration can be found [on the Wolfram Demonstrations Project](http:/demonstrations.wolfram.com/SuperdenseCoding/).
+- Superdense coding protocol is described in Nielsen & Chuang, section 2.3 (pp. 97-98).
+
+We split the superdense coding protocol into several steps, following the description in the [Wikipedia article](https://en.wikipedia.org/wiki/Superdense_coding):
+
+- Preparation (creating the entangled pair of qubits that are sent to Alice and Bob).
```

Comment:
Using Alice and Bob here is common, but it also implies that the learner already read the other sources. I would rename them to sender and receiver to make it more descriptive.

Reply:
Thanks for the suggestion. We were following this notation in previous kata. I added a note about Alice being the sender and bob being the receiver. Let me know if that works, or should I go more descriptive Alice(sender), Bob(receiver)?

Personally speaking, I like Alice(typical sender), Bob(typical receiver) and Charlie/Eve(Evesdropper). This is because I find it to be more cognitive distinguishable vs sender/receiver/evesdropper.